04-call-summaries/llm.py

```diff
@@ -0,0 +1,67 @@
+import dotenv
+
+from langchain_community.llms.ollama import Ollama
+import google.generativeai as genai
+from openai import OpenAI
+import os
+
+dotenv.load_dotenv()
+
+
+def get_transcript(file_path="./transcript.txt"):
+    file = open(file_path)
+    content = file.read()
+    return content
+
+
+def ollama_client(
+    model_name=None,
+    prompt=None,
+    callbacks=None,
+    settings=None,
+):
+    if not settings:
+        settings = {
+            # "temperature": 0.1,
+            # "system": "",
+            # "template": "",
+            # See https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/llms/ollama.py
+            "stop": None
+        }
+
+    print("LLM settings:", model_name, settings)
+    # To connect via another URL: Ollama(base_url='http://localhost:11434', ...)
+    return Ollama(model=model_name, callbacks=callbacks, **settings).invoke(prompt)
+
+
+def google_gemini_client(
+    model_name="gemini-pro",
+    prompt=None,
+    settings={},
```

Comment:
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/dangerous-default-value.html
Install pylint plugin for VSCode to detect such lints.
```suggestion
    settings=None,
```
Also see line 23 above.